<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="5" tests="12" time="174.803">
 <testsuite name="Verify validity of URL" timestamp="2018-08-08T12:53:01" hostname="localhost" time="30.872" errors="0" tests="1" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify validity of URL" name="should check if the url redirected the user to Snap Login page" time="30.862" />
 </testsuite>
 <testsuite name="Verify login functionality with valid credentials" timestamp="2018-08-08T12:53:31" hostname="localhost" time="64.477" errors="0" tests="4" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify login functionality with valid credentials" name="should enter username and password in Login Page" time="6.797" />
  <testcase classname="Verify login functionality with valid credentials" name="should verify and enter details for Slack Page" time="27.618" />
  <testcase classname="Verify login functionality with valid credentials" name="should enter details on organisational login page" time="13.597" />
  <testcase classname="Verify login functionality with valid credentials" name="should verify SNAP home page" time="16.464" />
 </testsuite>
 <testsuite name="Verify the login functionality with empty username" timestamp="2018-08-08T12:54:36" hostname="localhost" time="5.249" errors="0" tests="1" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify the login functionality with empty username" name="should not login the user and popup should be displayed" time="5.248" />
 </testsuite>
 <testsuite name="Verify the login functionality with invalid username" timestamp="2018-08-08T12:54:41" hostname="localhost" time="12.484" errors="0" tests="1" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify the login functionality with invalid username" name="should not login the user and error message should be displayed" time="12.484" />
 </testsuite>
 <testsuite name="Verify the login functionality with empty password" timestamp="2018-08-08T12:54:54" hostname="localhost" time="4.779" errors="0" tests="1" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify the login functionality with empty password" name="should not login the user and popup should be displayed" time="4.778" />
 </testsuite>
 <testsuite name="Verify the login functionality with invalid password" timestamp="2018-08-08T12:54:58" hostname="localhost" time="13.167" errors="0" tests="1" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify the login functionality with invalid password" name="should not login the user and error message should be displayed" time="13.167" />
 </testsuite>
 <testsuite name="Verify the login functionality with invalid credentials" timestamp="2018-08-08T12:55:12" hostname="localhost" time="11.959" errors="0" tests="1" skipped="0" disabled="0" failures="0">
  <testcase classname="Verify the login functionality with invalid credentials" name="should not login the user and error message should be displayed" time="11.959" />
 </testsuite>
 <testsuite name="Verify the login functionality with empty credentials" timestamp="2018-08-08T12:55:24" hostname="localhost" time="13.509" errors="0" tests="1" skipped="0" disabled="0" failures="2">
  <testcase classname="Verify the login functionality with empty credentials" name="should not login the user and error message should be displayed" time="13.509">
   <failure type="exception" message="Failed: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)"><![CDATA[WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)
    at Object.checkLegacyResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebElement.sendKeys()
    at thenableWebDriverProxy.schedule (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at WebElement.schedule_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
    at WebElement.sendKeys (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2174:19)
    at actionFn (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:89:44)
    at Array.map (<anonymous>)
    at actionResults.getWebElements.then (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:461:65)
    at ManagedPromise.invokeCallback_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)Error
    at ElementArrayFinder.applyAction_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as sendKeys] (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as sendKeys] (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at LoginPage.enterEmptyCredentials (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\pages\LoginPage.js:110:57)
    at UserContext.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:123:25)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it("should not login the user and error message should be displayed") in control flow
    at UserContext.<anonymous> (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:64:48
    at ControlFlow.emit (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
    at ControlFlow.shutdown_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
    at shutdownTask_.MicroTask (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2599:53)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:122:9)
    at addSpecsToSuite (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:115:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)]]>
   </failure>
   <failure type="exception" message="WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)"><![CDATA[WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)
    at Object.checkLegacyResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at UserContext.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\protractor.conf.js:31:21)
    at attempt (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9]]>
   </failure>
  </testcase>
 </testsuite>
 <testsuite name="Verify user navigation on Contact the Helpdesk link" timestamp="2018-08-08T12:55:37" hostname="localhost" time="18.307" errors="0" tests="1" skipped="0" disabled="0" failures="3">
  <testcase classname="Verify user navigation on Contact the Helpdesk link" name="should redirect user to helpdesk page" time="18.307">
   <failure type="exception" message="Failed: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)"><![CDATA[WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)
    at Object.checkLegacyResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.navigate().to(https://snap-nydailynews-stage.tribdev.com/)
    at thenableWebDriverProxy.schedule (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at Navigation.to (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1133:25)
    at thenableWebDriverProxy.get (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:988:28)
    at ProtractorBrowser.get (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\browser.js:655:32)
    at BasePage.navigateToURL (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\pages\BasePage.js:5:17)
    at UserContext.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:132:24)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
From: Task: Run beforeEach in control flow
    at UserContext.<anonymous> (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:129:9)
    at addSpecsToSuite (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:128:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)]]>
   </failure>
   <failure type="exception" message="Failed: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)"><![CDATA[WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)
    at Object.checkLegacyResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.findElements(By(xpath, //p[@id='header-help-text-links']/a))
    at thenableWebDriverProxy.schedule (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.findElements (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1048:19)
    at ptor.waitForAngular.then (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:159:44)
    at ManagedPromise.invokeCallback_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at process._tickCallback (internal/process/next_tick.js:68:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at LoginPage.contactHelpdesk (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\pages\LoginPage.js:115:63)
    at UserContext.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:136:25)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it("should redirect user to helpdesk page") in control flow
    at UserContext.<anonymous> (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
    at Function.next.fail (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4274:9)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:64:48
    at ControlFlow.emit (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
    at ControlFlow.shutdown_ (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:135:9)
    at addSpecsToSuite (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\src\test\test_spec\login_spec.js:128:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)]]>
   </failure>
   <failure type="exception" message="WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)"><![CDATA[WebDriverError: chrome not reachable
  (Session info: chrome=67.0.3396.99)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 6.3.9600 x86_64)
    at Object.checkLegacyResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at UserContext.<anonymous> (C:\Users\adalakoti\Music\SNAP_NEW\snap-master\protractor.conf.js:31:21)
    at attempt (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\satya\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>